tests/fast_tests/test_scenario_base_class.py

```diff
@@ -823,44 +820,32 @@ class TestNextPrevEdge(unittest.TestCase):
     the next edge)
     """
 
-    def test_next_edge_internal_links(self):
+    def test_next_prev_edge_figure_eight(self):
         """
-        Tests the next_edge() method in the presence of internal links.
+        Tests the next_edge() and prev_edge() methods for the figure eight.
         """
         env, scenario = figure_eight_exp_setup()
+
         next_edge = env.k.scenario.next_edge("bottom", 0)
         expected_next_edge = [(':center_1', 0)]
-
         self.assertCountEqual(next_edge, expected_next_edge)
 
-    def test_prev_edge_internal_links(self):
-        """
-        Tests the prev_edge() method in the presence of internal links.
-        """
-        env, scenario = figure_eight_exp_setup()
         prev_edge = env.k.scenario.prev_edge("bottom", 0)
         expected_prev_edge = [(':bottom_0', 0)]
-
         self.assertCountEqual(prev_edge, expected_prev_edge)
 
-    def test_next_edge_no_internal_links(self):
+    def test_next_prev_edge_ring_road(self):
```

Comment:
I think this could be named slightly better as it is it reads like it tests the "next previous edge" method. Maybe "test_edge_finding"?